sys/include/net/nanocoap.h

```diff
@@ -568,6 +579,26 @@ static inline unsigned coap_get_ver(const coap_pkt_t *pkt)
     return (pkt->hdr->ver_t_tkl & 0x60) >> 6;
 }
 
+/**
+ * @brief   Get the size of the extended Token length field
+ *          (RFC 8974)
+ *
+ * @param[in]   pkt   CoAP packet
+ *
+ * @returns     CoAP version number
+ */
+static inline uint8_t coap_hdr_tkl_ext_len(const coap_hdr_t *hdr)
+{
+    switch (hdr->ver_t_tkl & 0xf) {
+    case 13:
+        return 1;
+    case 14:
+        return 2;
+    default:
```

Comment:
I think it would be best to explicit let `15` fall through to default. That way the code would resemble the RFC a bit closer.

Reply:
15 could also be an `assert(0)` as it should be ruled out before this function is called :thinking: